sweep/sweeper.go

```diff
@@ -37,6 +37,7 @@ func (s *Sweeper) CreateSweepTx(
 	sweepTx.AddTxIn(&wire.TxIn{
 		PreviousOutPoint: htlcOutpoint,
 		SignatureScript:  htlc.SigScript,
+		Sequence:         htlc.Sequence(),
```

Comment:
What about the timeout sweep for loop in? I think that will now also get a sequence of 1

Reply:
fixed